tests/lib/SimpleSAML/ConfigurationTest.php

```diff
@@ -97,6 +104,9 @@ public function testHasValueOneOf() {
      * Test SimpleSAML_Configuration::getBaseURL()
      */
     public function testGetBaseURL() {
+
+        // Need to set a configuration file because the SSP Logger attempts to load it.
+        putenv('SIMPLESAMLPHP_CONFIG_DIR=tests/config/no-options');
```

Comment:
OK. I'll go through and change my use `putenv('SIMPLESAMLPHP_CONFIG_DIR=tests/config/no-options')` to this.